doc.go

```diff
@@ -315,36 +324,37 @@ Start the custom router:
 
 Proxy Package Used Individually
 
-The Run function in the root skipper package starts its own listener and
-doesn't provide the best composability. The proxy package, however,
-provides a standard http.Handler, so it is possible to use it in a more
-complex solution as a building block for routing.
+The 'Run' function in the root skipper package starts its own listener
+but it doesn't provide the best composability. The proxy package,
+however, provides a standard http.Handler, so it is possible to use it
+in a more complex solution as a building block for routing.
 
 
 Logging and Metrics
 
-Skipper provides detailed logging about unexpected failures, access logs
-in the Apache combined log format. If set up so, Skipper also collects
-detailed performance metrics, and exposes them on a separate listener
-endpoint for pulling snapshots. For more details, see the documentation
-of the logging and metrics subdirectories.
+Skipper provides detailed logging of failures, and access logs in Apache
+log format. When set up, Skipper also collects detailed performance
+metrics, and exposes them on a separate listener endpoint for pulling
+snapshots.
+
+For details, see the 'logging' and 'metrics' packages documentation.
 
 
 Performance Considerations
 
-While the real life performance of the router depends on the environment
-and the used filters, in ideal circumstances and without filters, the
-largest time factor is the route lookup. Skipper can scale to thousands
-of routes, with logarithmic performance degradation. However, this comes
-at the cost of memory consumption, due to storing the whole lookup tree
-in a single structure.
+The router's real life performance depends on the environment and on the
+used filters. Under ideal circumstances and without filters the biggest
+time factor is the route lookup. Skipper is able to scale to thousands
+of routes with logarithmic performance degradation. However, this comes
+at the cost of increased memory consumption, due to storing the whole
+lookup tree in a single structure.
 
 Benchmarks for the tree lookup can be run by:
 
     go test github.com/zalando/skipper/routing -bench=Tree
 
-In case of more agressive scaling, depending on the available memory,
-cascading multiple skipper instances based on segments of routes can be
-the preferrable approach.
+In case of more agressive scaling is needed, depending on the available
```

Comment:
In case more aggressive scale is needed, it is possible to setup Skipper in a cascade model, with multiple Skipper instances for specific route segments.